Dockerfile

```diff
@@ -28,4 +28,8 @@ RUN ln -s /usr/lib/chromium/chromedriver /usr/local/bin/
 
 USER webdriver
 
-CMD ["chromedriver", "--url-base=/wd/hub", "--port=4444", "--whitelisted-ips="]
+ENTRYPOINT ["entrypoint", "chromedriver"]
+
+CMD ["--port=4444", "--whitelisted-ips="]
```

Comment:
is `--url-base=/wd/hub` not needed anymore?

Reply:
https://github.com/blueimp/chromedriver/commit/6637aa8fed968ef4957a148ae94c3648aa8f09c6